system/hardware/tici/tests/test_power_draw.py

```diff
@@ -20,14 +23,15 @@ class Proc:
   rtol: float = 0.05
   atol: float = 0.12
   warmup: float = 6.
+  msgs: List[str] = field(default_factory=list)
```

Comment:
wouldn't set a default here, otherwise it's easy to skip the coverage of the message checks